.changeset/beige-jokes-add.md

```diff
@@ -0,0 +1,22 @@
+---
+'backend': major
+'app': major
+---
+
+Update the proxy object to include the endpoint property in `app-config.yaml`
```

Comment:
@schultzp2020 @tumido , this is a breaking change. Should we mention this here, or it's better with the Helm Chart release as customers will have to update their config files?